manifests/params.pp

```diff
@@ -20,7 +20,11 @@
         'Ubuntu' => if versioncmp($facts['os']['release']['major'], '22.04') >= 0 { '/usr/bin/named-checkconf' } else { '/usr/sbin/named-checkconf' },
         default  => '/usr/sbin/named-checkconf',
       }
-      $sysconfig_file     = '/etc/default/bind9'
+      $sysconfig_file     = $facts['os']['name'] ? {
+        'Debian' => if versioncmp($facts['os']['release']['major'], '10') >= 0 { '/etc/default/bind9' } else { '/etc/default/named' },
+        'Ubuntu' => if versioncmp($facts['os']['release']['major'], '18.04') >= 0 { '/etc/default/bind9' } else { '/etc/default/named' },
```

Comment:
Same here. I think your logic is incorrect and should be:
```suggestion
        'Ubuntu' => if versioncmp($facts['os']['release']['major'], '20.04') >= 0 { '/etc/default/named' } else { '/etc/default/bind9' },
```

Reply:
pushed a change to fix this.